Changelog.md

```diff
@@ -73,6 +73,7 @@ Breaking Changes:
  * Syntax Checker: Strictly require visibility specifier for functions. This was already the case in the experimental 0.5.0 mode.
  * Syntax Checker: Disallow unary ``+``. This was already the case in the experimental 0.5.0 mode.
  * Syntax Checker: Disallow single statement variable declaration inside if/while/for bodies that are not blocks.
+ * Syntax Checker: Improved error message for lookup in function types.
```

Comment:
This is not a breaking change. Please move it to the "compiler features" section.